recipes/libproperties/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibpropertiesConan(ConanFile):
+    name = "libproperties"
+    license = "Apache-2.0"
+    homepage = "https://github.com/tinyhubs/libproperties"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "libproperties is a library to parse the Java .properties files. It was writen in pure C. And fully compatible with the Java .properties file format."
+    topics = ("properties", "java", "pure-c")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False],}
+    default_options = {"shared": False, "fPIC": True,}
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "sources"
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        source_dir="libproperties-{}".format(self.version)
+        os.rename(source_dir, self._source_subfolder)
+        tools.replace_in_file(source_dir, 
```

Comment:
Please create a cmake wrapper file, wherein you do:
```cmake
cmake_minimum_required(VERSION 2.9.18)
project(cmake_wrapper C)
include(conanbuildinfo.cmake)
conan_basic_setup()
add_subdirectory(source_subfolder)
```
and add `CMakeLists.txt` to `exports_sources`.

Reply:
but why?

Reply:
Then you don't need to patch

Reply:
It's also needed on MSVC.
The wrapper script makes sure that the C runtime is configured correctly.
MSVC has MT/MD/MTd/MDd runtime: static, shared, debug static, debug shared.

Reply:
It's also required on linux to set some of the GNU flags

Reply:
Ah, I guess you mean libstdc++ vs libstdc++11?